addons/stock/tests/test_robustness.py

```diff
@@ -71,19 +71,23 @@ def test_location_usage(self):
         consistent with the `reserved_quantity` on the quants.
         """
         # change stock usage
-        self.stock_location.scrap_location = True
+        test_stock_location = self.env['stock.location'].create({
+            'name': "Test Location",
+            'location_id': self.stock_location.id,
+        })
+        test_stock_location.scrap_location = True
```

Comment:
put this directly in create()

Reply:
That was already merged in the bugfix task :fearful: